v03_pipeline/lib/misc/callsets.py

```diff
@@ -6,13 +6,14 @@
 from v03_pipeline.lib.paths import remapped_and_subsetted_callset_path
 
 
-def get_callset_ht(
+def get_callset_ht(  # noqa: PLR0913
     reference_genome: ReferenceGenome,
     dataset_type: DatasetType,
     callset_paths: list[str],
     project_guids: list[str],
     project_remap_paths: list[str],
     project_pedigree_paths: list[str],
+    imputed_sex_paths: list[str] | None,
```

Comment:
so we're storing callset-wide metric tables, yeah? Seems like the right implementation. Thoughts on some new `gs://seqr-loading-temp/v03/GRCh38/SNV_INDEL/callset_metrics/` ?

Reply:
we do store the "sex_check" table, which is pretty much just this now.  

in general all the "paths" used here are considered "arbitrary", so we don't have a way of computing them from the inputs in the way we do for the rest of the pieces of the pipeline.